logstash-core/api/lib/app.rb

```diff
@@ -0,0 +1,38 @@
+# encoding: utf-8
+require "json"
+require "app_helpers"
+require "app/service"
+require "app/command_factory"
+
+module LogStash::Api
+  class BaseApp < ::Sinatra::Application
+
+    attr_reader :factory
+
+    if settings.environment == :production
+      ::Logger.class_eval { alias :write :'<<' }
+      access_log    = ::File.join(::File.dirname(::File.expand_path(__FILE__)),'../../..', 'access.log')
+      access_logger = ::Logger.new(access_log)
+      error_logger  = ::File.new(::File.join(::File.dirname(::File.expand_path(__FILE__)),'../../..','error.log'),"a+")
+      error_logger.sync = true
```

Comment:
I think we should use our internal logger here (cabin) we could probably ignore the access log unless we run in debug mode?